modules/serverpod_auth/serverpod_auth_server/lib/src/business/config.dart

```diff
@@ -155,6 +159,7 @@ class AuthConfig {
     this.onUserUpdated,
     this.sendPasswordResetEmail,
     this.sendValidationEmail,
+    this.validationCodeLength = 8,
```

Comment:
Discussion: Should we safeguard against setting the value to 0 or a negative value? Or what makes sense in that case?

Reply:
It might be good to add a suggestion for at least three digits with this motivation in the documentation. But I think, ultimately, the framework should not be in your way if you would like to use a single-digit code.

Reply:
True :) But this sounds like an elegant solution to me. 

Safeguarding against `0` or `negative` 

Recommending / informing in the docs